### ISSUE TITLE
Wire Desktop app uses an executable stack in Version 3.0.2816  

### ISSUE THREAD

**Issue by 0x00rick** [wireapp/wire-desktop#1464]
### Info
Wire-Desktop version: Version 3.0.2816  
Operating system: Ubuntu 18.04 XFCE

The prebuilt packages for Linux (deb/rpm) contain a version of Wire-Desktop marks the stack as Read,Write,eXecute for each wire-desktop process.  

### Reproduction:

Install the x64 wire-desktop app from e.g.: the provided .deb package and start it.
Then execute the following:

```
for i in $(pidof wire-desktop); do cat /proc/$i/maps | grep stack; done 
7fff94ef4000-7fff94f15000 rwxp 00000000 00:00 0                          [stack]
7fff94ef4000-7fff94f15000 rwxp 00000000 00:00 0                          [stack]
7fff94ef4000-7fff94f15000 rwxp 00000000 00:00 0                          [stack]
7fff1e339000-7fff1e35a000 rwxp 00000000 00:00 0                          [stack]
7fff94ef4000-7fff94f15000 rwxp 00000000 00:00 0                          [stack]
7ffda91dc000-7ffda91fd000 rwxp 00000000 00:00 0                          [stack]
```

### What is the expected result?

The stack should not be executable!

### What is the actual result?

The stack is executable!

**Comment by 0x00rick** [comment]
@ffflorian Good to hear you quickly upgraded to the non affected electron release!

**Comment by mythsunwind** [comment]
This issue is fixed with release Linux release 3.2.2840 on debian repository and https://github.com/wireapp/wire-desktop/releases

```
$ for i in $(pidof wire-desktop); do cat /proc/$i/maps | grep stack; done
7fff3240b000-7fff3242f000 rw-p 00000000 00:00 0                          [stack]
7fff3240e000-7fff3242f000 rw-p 00000000 00:00 0                          [stack]
7fff3240e000-7fff3242f000 rw-p 00000000 00:00 0                          [stack]
7ffd744d2000-7ffd744f3000 rw-p 00000000 00:00 0                          [stack]
```
